### PR TITLE
Bug fix - #1837

### PR DESCRIPTION
**Pull request by achinthagunasekara**
Bug fix reported in https://github.com/kubernetes/autoscaler/issues/1862

### CODE REVIEW

**Comment by achinthagunasekara** [comment]
cc @frobware

File: cluster-autoscaler/fix_gopath.sh [hunk 1]
```diff
@@ -11,7 +11,8 @@ for item in $to_fix; do
    mkdir "${GOPATH}/src/k8s.io/${item}"
    cd "${GOPATH}/src/k8s.io/${item}"
    git init
-   cp -R "${GOPATH}/src/k8s.io/kubernetes/staging/src/k8s.io/${item}/*" ./
+   # shellcheck disable=SC2086
+   cp -R ${GOPATH}/src/k8s.io/kubernetes/staging/src/k8s.io/${item}/* ./
```
Comment (frobware) [inline cluster-autoscaler/fix_gopath.sh]: There are other places below this that need the same treatment.

Comment (achinthagunasekara) [reply cluster-autoscaler/fix_gopath.sh]: All fixed :)

File: cluster-autoscaler/fix_gopath.sh [hunk 2]
```diff
@@ -27,7 +27,8 @@ done
 
 echo Overriding AKS API
 mkdir -p "${GOPATH}/src/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice"
-cp "${GOPATH}/src/k8s.io/autoscaler/cluster-autoscaler/_override/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice/*" "${GOPATH}/src/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice/"
+# shellcheck disable=SC2086
+cp ${GOPATH}/src/k8s.io/autoscaler/cluster-autoscaler/_override/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice/* "${GOPATH}/src/github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice/"
```
Comment (frobware) [inline cluster-autoscaler/fix_gopath.sh]: There's still another after `echo Overriding Azure autorest`

Comment (achinthagunasekara) [reply cluster-autoscaler/fix_gopath.sh]: Oops, missed this one. Fixed :)

**Comment by achinthagunasekara** [comment]
@mwielgus Done :)

**Pull request merged**